packages/fields/src/types/Checkbox/Implementation.js

```diff
@@ -25,20 +25,16 @@ export class Checkbox extends Implementation {
   }
 }
 
-const CommonCheckboxInterface = superclass =>
-  class extends superclass {
-    getQueryConditions(dbPath) {
-      return this.equalityConditions(dbPath);
-    }
-  };
-
-export class MongoCheckboxInterface extends CommonCheckboxInterface(MongooseFieldAdapter) {
+export class MongoCheckboxInterface extends MongooseFieldAdapter {
```

Comment:
Why the class refactor here?

Reply:
Didn't seem worth the complexity to avoid repeating 3 lines. The new version achieves the same thing, is easier to understand, creates less variables/classes and is shorter/less code.

I'm not that fussed if you want to revert it though.